main.tf

```diff
@@ -10,7 +6,15 @@ resource "aws_prometheus_workspace" "this" {
   count = var.create && var.create_workspace ? 1 : 0
 
   alias = var.workspace_alias
-  tags  = var.tags
+
+  dynamic "logging_configuration" {
```

Comment:
this should be:

```hcl
dynamic "logging_configuration" {
    for_each = length(var.logging_configuration) > 0 ? [var.logging_configuration] : []

    content {
      log_group_arn = logging_configuration.value.log_group_arn
    }
  }
```

and the variables will need to be updated to match

Reply:
Fixed in new commit.

Thanks a ton !